spark/src/test/java/org/apache/iceberg/spark/source/TestIcebergSourceHiveTables.java

```diff
@@ -147,32 +131,7 @@ public synchronized void testHiveEntriesTable() throws Exception {
       Table table = catalog.createTable(tableIdentifier, SCHEMA, PartitionSpec.unpartitioned());
       Table entriesTable = catalog.loadTable(TableIdentifier.of("db", "entries_test", "entries"));
 
-      List<SimpleRecord> records = Lists.newArrayList(new SimpleRecord(1, "1"));
-
-      Dataset<Row> inputDf = spark.createDataFrame(records, SimpleRecord.class);
-      inputDf.select("id", "data").write()
-          .format("iceberg")
-          .mode("append")
-          .save(tableIdentifier.toString());
-
-      table.refresh();
-
-      List<Row> actual = spark.read()
-          .format("iceberg")
-          .load("db.entries_test.entries")
-          .collectAsList();
-
-      Assert.assertEquals("Should only contain one manifest", 1, table.currentSnapshot().manifests().size());
-      InputFile manifest = table.io().newInputFile(table.currentSnapshot().manifests().get(0).path());
-      List<GenericData.Record> expected;
-      try (CloseableIterable<GenericData.Record> rows = Avro.read(manifest).project(entriesTable.schema()).build()) {
-        expected = Lists.newArrayList(rows);
-      }
-
-      Assert.assertEquals("Entries table should have one row", 1, expected.size());
-      Assert.assertEquals("Actual results should have one row", 1, actual.size());
-      TestHelpers.assertEqualsSafe(entriesTable.schema().asStruct(), expected.get(0), actual.get(0));
-
+      super.testEntriesTable(table, entriesTable, tableIdentifier.toString(), "db.entries_test.entries");
```

Comment:
The Hadoop table tests use before/after methods to create and clean up the table. Could this use an after method that calls `DROP TABLE IF EXISTS` to avoid the try/finally in each method?